nucliadb_models/nucliadb_models/text.py

```diff
@@ -87,6 +89,7 @@ class PushTextFormat(TextFormatValue, Enum):  # type: ignore
     HTML = 1
     MARKDOWN = 2
     RST = 3
+    JSON = 4
```

Comment:
@bloodbare make sure this is expected in the processor

Reply:
+1